tests/Task.test.ts

```diff
@@ -713,6 +714,15 @@ describe('toggle done', () => {
         StatusRegistry.getInstance().resetToDefaultStatuses();
     });
 
+    beforeEach(() => {
+        jest.useFakeTimers();
+    });
+
+    afterEach(() => {
+        jest.useRealTimers();
+        resetSettings();
+    });
+
```

Comment:
In a later change I will move almost all the `afterEach()` content to the top of the file, outside all `describe` blocks.